contracts/base/Settings.sol

```diff
@@ -86,6 +86,13 @@ contract Settings is Pausable, SettingsInterface {
      */
     mapping(address => bool) public lendingPoolPaused;
 
+    /**
+        @notice Contains minimum version for each node component.
+
+        i.e.: "web2" => "1234500" represents "web2" => "01.23.45.00"
+     */
+    mapping(bytes32 => uint32) public minimumNodeRequirements;
```

Comment:
I would change minimumNodeRequirements to componentVersions. So, it matches with the function (update/create/remove) names.